src/main/kotlin/ee/carlrobert/codegpt/codecompletions/CodeGPTInlineCompletionProvider.kt

```diff
@@ -78,6 +78,18 @@ class CodeGPTInlineCompletionProvider : InlineCompletionProvider {
         private val completed: (String) -> Unit
     ) : CompletionEventListener<String> {
 
+        private var isStreaming = false
+
+        override fun onMessage(message: String?, eventSource: EventSource?) {
+            if (message != null) {
+                isStreaming = true
```

Comment:
Can we get rid of this class variable?

This should be enough:
```kt
if (message.contains('\n') && eventSource != null) {
    eventSource.cancel()
}
```

Reply:
isStreaming and `completed(message.takeWhile { it != '\n' })` is for streaming completion to reduce user waiting anxiety
I think we can remove isStreaming variables and onComplete() method assuming all LLM services support streaming responses

Reply:
isStreaming and onComplete() have been removed